letsencrypt/cli.py

```diff
@@ -609,7 +622,7 @@ class HelpfulArgumentParser(object):
     """
 
     # Maps verbs/subcommands to the functions that implement them
-    VERBS = {"auth": auth, "config_changes": config_changes,
+    VERBS = {"auth": obtaincert, "certonly": obtaincert, "config_changes": config_changes,
```

Comment:
nit: The alias `auth` is included here but not `everything`. I think we should have the same behavior for both aliases. The only behavior that I'm aware of that is affected here is whether you can get help using the alias (e.g. `letsencrypt --help auth`).

I don't have a strong opinion, but I think aliases should not be included. The aliases are not shown in the help output and the distinction between `letsencrypt --help all` and `letsencrypt --help everything` is somewhat confusing.